provisioning/roles/docker/tasks/main.yml

```diff
@@ -0,0 +1,53 @@
+---
+- include_vars: ../../common_vars.yml
+
+- name: Setup the docker repository and install docker
+  become: yes
+  shell: |
+    apt-get update
+    apt-get install apt-transport-https ca-certificates curl software-properties-common --yes
+    curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+    add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
+    apt-get update && apt-get install docker-ce --yes
+
+- name: Download docker-compose
+  become: yes
+  shell: curl -L https://github.com/docker/compose/releases/download/1.21.2/docker-compose-$(uname -s)-$(uname -m) -o /usr/local/bin/docker-compose
+
+- name: Apply executable permissions to the docker-compose binary
+  become: yes
+  shell: chmod +x /usr/local/bin/docker-compose
+
+- name: Copy docker-compose.yml
+  copy: src={{ item.src }} dest={{ item.dest }} owner=ubuntu group=ubuntu mode=0644
+  with_items:
+    - { src: '../files/docker-compose.yml', dest: '/home/ubuntu/snowplow/' }
+
+- name: Ensure file permissions
+  become: yes
+  shell: chown ubuntu:ubuntu -R /home/ubuntu/snowplow && chmod 755 -R /home/ubuntu/snowplow
```

Comment:
why is this necessary, can we add a comment?

Reply:
I forgot to delete it, I used that for test purposes. 4th role already does that.